src/mon/MonCap.cc

```diff
@@ -173,25 +173,6 @@ void MonCapGrant::expand_profile(int daemon_type, const EntityName& name) const
     return;
   }
 
-  switch (daemon_type) {
-  case CEPH_ENTITY_TYPE_MON:
-    expand_profile_mon(name);
-    return;
-  case CEPH_ENTITY_TYPE_MGR:
-    expand_profile_mgr(name);
-    return;
-  }
-}
-
-void MonCapGrant::expand_profile_mgr(const EntityName& name) const
-{
-  if (profile == "crash") {
-    profile_grants.push_back(MonCapGrant("crash post"));
-  }
```

Comment:
I don't see that this got added to the new MgrCap profile expansion?

Reply:
Should be here: https://github.com/ceph/ceph/pull/30912/files#diff-4c545258ced8b2b154aa236dac3ecf50R154